app/views/speakers/_form.html.erb

```diff
@@ -11,71 +11,72 @@
     </div>
   <% end %>
 
-  <h2>Speaker</h2>
-
-  <div class="field">
-    <%= form.label :name %>
-    <%= form.text_field :name, id: :speaker_name %>
-  </div>
-
-  <div class="field">
-    <%= form.label :picture %>
-    <%= form.file_field :picture %>
-  </div>
-
-  <div class="field">
-    <%= form.label :email %>
-    <%= form.text_field :email %>
-  </div>
-
-  <div class="field">
-    <%= form.label :company %>
-    <%= form.text_field :company %>
-  </div>
-
-  <div class="field">
-    <%= form.label :biography %>
-    <%= form.text_area :biography %>
-  </div>
+  <div class="form">
+    <h2>Palestrante</h2>
+    <div class="field">
+      <%= form.label :nome %>
+      <%= form.text_field :name, id: :speaker_name, class: :input %>
+    </div>
 
-  <div class="field">
-    <%= form.label :website %>
-    <%= form.text_field :website %>
-  </div>
+    <div class="field">
+      <%= form.label :foto %>
+      <%= form.file_field :picture, class: :inputpicture %>
```

Comment:
Its better to pass the class as a string. `<%= form.file_field :picture, class: 'inputpicture' %>`